### ISSUE TITLE
Joint font style and text effect transfer

### ISSUE THREAD

**Issue by XiaoYangon** [williamyang1991/TET-GAN#3]
Hello, your project is really meaningful. In the paper, you mention jointfont style and text effect transfer?  how can this be realized? Looking forward to your reply.

**Comment by williamyang1991** [comment]
You should just prepare a font dataset and train another TET-GAN on font dataset. 
Let TET-GAN trained on font be G1, and trained on text effects be G2.
To transfer the text effects and font, first use G2 to destylize your reference style image Y and obtain the glyph image X.
Then use G1 to stylize your text image I with X to obtain the font transfer result I_X;
Finally, use G2 to stylize I_X with Y to obtain the joint font and style transfer result.

**Comment by XiaoYangon** [comment]
@williamyang1991 Thank you for your reply. There are several questions to ask: are G1 and G2 based on the current TET-GAN model training? Are these two model independently trained? Do I need to make any changes to the model and loss function in order to implement G1 and G2?

**Comment by williamyang1991** [comment]
There is no difference between G1 and G2. They are trained independently.
